app/views/pages/resources.html.erb

```diff
@@ -112,6 +112,8 @@
           <li><%= link_to 'DD1351-2 - Travel Voucher', 'http://www.esd.whs.mil/Portals/54/Documents/DD/forms/dd/dd1351-2.pdf', class: "usa-external_link" %>: This form needs to be completed and submitted to your local finance office at the completion of your move so you can claim all your reimbursable expenses that you are entitled to during your move. Some finance offices have online systems for this form in place, however it’s listed here if needed.</li>
           <li><%= link_to 'Sample Inconvenience Claim Letter', asset_path('claim.pdf'), download: 'claim.pdf' %>: Included is a sample inconvenience claim letter to use as an example. For more information on an Inconvenience claim, please review this <%= link_to 'Frequently Asked Question', faqs_path %>.</li>
           <li><%= link_to 'PPM Submission Checklist', asset_path('ppm_checklist.pdf'), download: 'ppm_checklist.pdf' %>: The Personally Procured Move checklist will help you gather all the required documents prior to submitting your PPM voucher to the personal property office.</li>
+          <li><%= link_to 'ATF Form 6 (Active Duty Service Members Only)', 'https://www.atf.gov/firearms/docs/form/form-6-part-2-application-and-permit-importation-firearms-ammunition-and', class: 'usa-external_link' %>: This form is used by Active Duty Military members to apply to permanently import firearms, ammunition and implements of war into the United States.</li>
+          <li><%= link_to 'ATF Form 6 (Civilians & Non-Active Duty Service Members)', 'https://www.atf.gov/firearms/docs/form/form-6-part-1-application-and-permit-importation-firearms-ammunition-and', class: 'usa-external_link' %>: This form is used by civilians and non-active duty service members apply to permanently import firearms, ammunition and implements of war into the United States.</li>
```

Comment:
I guess Rails automatically inserts the ampersand entity?

Reply:
Correct!